content/changes.md

```diff
@@ -8,6 +8,46 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## January, 2024 
+
+### Features 
+ 
+- **Learnosity Partnership Integration** 
+
+    [Learnosity](https://learnosity.com/) has officially announced our partnership and integration into their platform to their custom base, via website, newsletter, and socials! 
+
+    This integration allows us to expand our reach into the L&D space, acting as an important add-on component within Learnosity's deep offering across many different learning domains. We've closed our first partner sale via iCEV, and already have other potential deals beginning to surface.       We will be presenting internally to their sales team later this month to further deepen our partnership and enable their sales team to bring more business to us.  
+
+    [Release Announcement](https://help.learnosity.com/hc/en-us/articles/14870489871133-2024-1-LTS-Release-Announcement)
+
+    [Partner Page](https://learnosity.com/company/partners/partners-directory/qualified/)
+
+
+### Improvements 
+
+- **Erlang 26** 
+
+    Erlang 26 is now supported within project challenges. 
+
+- **Elixir 1.15** 
+
+    Elixir 1.15 is now supported within project challenges.
+
+- **Add deprecated language version labels** 
+
+    A new “deprecated” label was added on old language versions in the challenge edit and detail views, visible to teams and content managers with the purpose of providing our users with more legibility of what are the available language versions to use. For further information please check       our [documentation](https://docs.qualified.io/creating-content/challenges/upgrading-language-versions/#upgrading-language-versions) 
+  
+- **Performance issue when loading "Portfolio" tab of report card** 
+
+    Users were experiencing heavy lag, especially with scrolls, once Portfolio tab was activated and the number of assessments results was exceeding 6 or more. This issue was successfully fixed.  
```

Comment:
In general, it's good to share bug fixes and performance improvements like this, but in this case, the portfolio is only available for teams with signals enabled. I don't see anything else in the product updates historically related to signals, so it may be something we skip just because it doesn't affect all customers.

Reply:
Yes, I wasn't sure about this because could have a bad impact over the fact we were able to solved a major performance issue. I'm going to remove it.

Reply:
I might be misunderstanding your response, but in general, it's good to be honest about solving issues--we don't want to hide bug fixes, no matter how large. Big companies like [OpenAI do postmortems for outages](https://openai.com/blog/march-20-chatgpt-outage).

So the reason not to share this is (potentially) only that it's a flagged feature that doesn't pertain to most of our customers.